utils/index.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -exo pipefail
+
+setup(){
+    # Generate a uuid
+    export UUID=${UUID:-$(uuidgen)}
+    # Elasticsearch Config
+    export ES_SERVER=$ES_SERVER
+    export ES_INDEX=$ES_INDEX
+    if [[ -z $PROW_JOB_ID ]]; then
+        export dag_id=${AIRFLOW_CTX_DAG_ID}
+        export execution_date=${AIRFLOW_CTX_EXECUTION_DATE}
+        export dag_run_id=${AIRFLOW_CTX_DAG_RUN_ID}
+        export ci="AIRFLOW"
+        printenv
+        # Get Airflow URL
+        export airflow_base_url="http://$(kubectl get route/airflow -n airflow -o jsonpath='{.spec.host}')"
+        # Setup Kubeconfig
+        export KUBECONFIG=/home/airflow/auth/config
+        curl -sS https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz oc
+        export PATH=$PATH:/home/airflow/.local/bin:$(pwd)
+        # ReleaseStream is piped in via environment variables
+        release_stream=${RELEASE_STREAM}
+    else
+        export ci="PROW"
+        export prow_base_url="https://prow.ci.openshift.org/view/gs/origin-ci-test/logs"
+    fi
+    # Get OpenShift cluster details
+    cluster_name=$(oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}') || echo "Cluster Install Failed"
+    cluster_version=$(oc version -o json | jq -r '.openshiftVersion') || echo "Cluster Install Failed"
+    network_type=$(oc get network.config/cluster -o jsonpath='{.status.networkType}') || echo "Cluster Install Failed"
+    platform=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}') || echo "Cluster Install Failed"
```

Comment:
Do we want to determine if the cluster is ROSA or not?, if so
```
  platformStatus:
    aws:
      region: us-west-2
      resourceTags:
      - key: User
        value: krishvoor
      - key: red-hat-clustertype
        value: rosa
      - key: red-hat-managed
        value: "true"
    type: AWS

```

Reply:
Maybe to be consistent w/ go-commons we should use the value @rsevilla87 mentions here? https://github.com/cloud-bulldozer/go-commons/blob/main/ocp-metadata/ocp-metadata.go#L61-L65 

I would think we should have platform be aws and maybe introduce a new field `cluster-type`, which if this is set use that value, if it isn't set `self-managed`

Reply:
Introduced new field for `cluster_type`.

Reply:
it would be good to introduce this in go-commons too -- to just be consistent.

Reply:
so, to be consistent with some of our other keys in ES -- or to attempt to be more consistent, we should use clusterType vs cluster_type.

Reply:
Changed as suggested to maintain keys consistent with our GO tools. Thank you!